src/solvers/LevenbergMarquardt.jl

```diff
@@ -207,7 +207,7 @@ function initialize_solver!(
     lms::LevenbergMarquardtState,
 ) where {mT<:AbstractManifold}
     get_objective(dmp).f(get_manifold(dmp), lms.residual_values, lms.p)
-    lms.X = get_gradient(dmp, lms.p)
+    get_gradient!(dmp, lms.X, lms.p, lms.jacF)
```

Comment:
```suggestion
    get_gradient!(dmp, lms.X, lms.p)
```

with the change above this should also be fine (again).

Reply:
(see above) please change this after adopting my remarks above.